Script/char.dat

```diff
@@ -5178,8 +5898,15 @@ floatingeye
   DefaultMana = 0;
   CanOpen = false;
   TotalVolume = 30000;
-  HostileReplies == "@Dd stares at you.";
-  FriendlyReplies == "@Dd stares at you.";
+  HostileReplies =
+  {
+    4,
+    "@Dd stares at you intently.",
+    "@Dd blinks.",
```

Comment:
How do you know? It could have some sort of an eyelid where you can't see it on the small sprite. :)